lms/djangoapps/course_api/tests/test_api.py

```diff
@@ -42,10 +41,20 @@ class CourseApiTestMixin(CourseApiFactoryMixin):
         'blocks_url': '/api/courses/v1/blocks/?course_id=edX%2Ftoy%2F2012_Fall',
     }
 
+    def verify_course(self, course, course_id=None):
+        """
+        Ensure that the returned course is the course we just created
+        """
+
+        if course_id is None:
+            course_id = self.expected_course_data['course_id']
+        # self.assertIsInstance(course, CourseOverview)
+        self.assertEqual(course_id, str(course.id))
```

Comment:
I moved the verification of other fields to test_serializers.py, but I can duplicate that here as well.  Will update.